runtime/near-vm-runner-standalone/contract/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "basic-contract"
+version = "0.1.0"
+authors = ["Near Inc <hello@nearprotocol.com>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { git = "https://github.com/nearprotocol/json", rev = "1f5779f3b0bd3d2a4b0b975abc46f3d3fe873331", features = ["no_floats"] }
+near-bindgen = "0.6.0"
+borsh = "*"
```

Comment:
Please specify a version. We sometimes have non-backwards compatible changes to borsh that are hard to debug through the contract.

Reply:
Done, was made this way to resolve rather tricky conflict between borsch in near-bindgen and one listed here resulting in cryptic errors.